src/App.vue

```diff
@@ -476,11 +486,29 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
 
   /** Gets account information (e.g. Premium account) and stores it. */
   private loadAccountInformation (): void {
-    if (sessionStorage.getItem(SessionStorageKeys.CurrentAccount)) {
-      const accountInfo = JSON.parse(sessionStorage.getItem(SessionStorageKeys.CurrentAccount))
+    const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+    if (currentAccount) {
+      const accountInfo = JSON.parse(currentAccount)
       this.setAccountInformation(accountInfo)
     }
   }
+
+  /** Fetches current user and updates Launch Darkly accordingly. */
+  private async loadCurrentUserLd (): Promise<any> {
+    try {
+      const response = await this.getCurrentUser()
```

Comment:
Not sure the impact here, but staff do not have a current user object to pull on, from my experience using there IDIR. Will this affect opening feature flagging up to staff?

Reply:
In my local testing, I do get a current user object... This is me logged in as a staff (IDIR) user:

![image](https://user-images.githubusercontent.com/10002889/92946057-ffff6780-f40a-11ea-9712-b30eacdddcca.png)

Reply:
And here's a BCSC user:

![image](https://user-images.githubusercontent.com/10002889/92946393-756b3800-f40b-11ea-938a-ac0a2be10534.png)

Reply:
It looks like this will work fine. Thoughts?

Reply:
O wow, interesting! I must have overlooked this. I was logging in with my IDIR and using our URL to directly access the EDIT UI - Correction in a new window after getting authenticated. I think that is why the Current User wasn't persisting there. I can certainly see it after login 🤦  Oops, but good to know!

Reply:
Can this conversation be resolved?